src/django_components/app_settings.py

```diff
@@ -118,6 +126,25 @@ def RELOAD_ON_TEMPLATE_CHANGE(self) -> bool:
     def TEMPLATE_CACHE_SIZE(self) -> int:
         return self.settings.get("template_cache_size", 128)
 
+    @property
+    def STATIC_FILES_ALLOWED(self) -> List[str]:
+        default_static_files = [
+            # Allow suffixes: .js, .css
+            r"\.(?:js|css)$",
```

Comment:
Should we not also allow images by default?

Reply:
Suggestion:

```python
default_static_files = [".css", ".gif", ".ico", ".jpeg", ".jpg", ".js", ".png", ".svg", ".webp"]
````

Reply:
Good question. We can add [common image types as defined in MDN](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#common_image_file_types).

As for fonts, at first my reaction was that these should be probably at the root of the project, outside of components. But one could have an "app" component that would hold fonts and such, so I can see that now. We can go with the ones listed here https://stackoverflow.com/q/30572159/9788634

Reply:
For the regex patterns, what do you say if we supported both? That preferred way would be to pass in strings, which would be evaluated as suffixes like `.html`. But one could also pass an instance of `re.Pattern` (AKA the result of `re.compile`) for more control.

Reply:
Updated, see the changes here https://github.com/EmilStenstrom/django-components/pull/652/files/766fd047711463a32f2910cf578dbfcb27b5cd9d..16b418d8d78e56b78a68f0a6f184ade8dc6daec4